ipn/ipnlocal/local.go

```diff
@@ -150,18 +150,19 @@ type LocalBackend struct {
 	shouldInterceptTCPPortAtomic syncs.AtomicValue[func(uint16) bool]
 
 	// The mutex protects the following elements.
-	mu             sync.Mutex
-	filterHash     deephash.Sum
-	httpTestClient *http.Client // for controlclient. nil by default, used by tests.
-	ccGen          clientGen    // function for producing controlclient; lazily populated
-	sshServer      SSHServer    // or nil, initialized lazily.
-	notify         func(ipn.Notify)
-	cc             controlclient.Client
-	ccAuto         *controlclient.Auto // if cc is of type *controlclient.Auto
-	machinePrivKey key.MachinePrivate
-	tka            *tkaState
-	state          ipn.State
-	capFileSharing bool // whether netMap contains the file sharing capability
+	mu                  sync.Mutex
+	filterHash          deephash.Sum
+	httpTestClient      *http.Client // for controlclient. nil by default, used by tests.
+	ccGen               clientGen    // function for producing controlclient; lazily populated
+	sshServer           SSHServer    // or nil, initialized lazily.
+	notify              func(ipn.Notify)
+	cc                  controlclient.Client
+	ccAuto              *controlclient.Auto // if cc is of type *controlclient.Auto
+	machinePrivKey      key.MachinePrivate
+	tka                 *tkaState
+	state               ipn.State
+	capFileSharing      bool // whether netMap contains the file sharing capability
+	capTailnetLockAlpha bool // whether netMap contains the tailnet lock alpha capability
```

Comment:
I kind of feel like `catpTailnetLock` would be less likely to go out of date in future if we keep that beyond the alpha timeframe. It's also shorter :-D

Reply:
Done.